src/mozillavpn.cpp

```diff
@@ -306,12 +306,17 @@ void MozillaVPN::initialize() {
 #endif
 
   m_private->m_captivePortalDetection.initialize();
+  logger.debug() << "Captive Portal Detection initialized";
```

Comment:
This is a lot of debug to leave behind in this file. I'd clean it up before merging.